docs/blog/newsletter-august-2017.rst

```diff
@@ -21,7 +21,7 @@ What resume advice is the *right* resume advice?
 
 For anyone currently job-hunting, this story will probably be pretty resonant. The question arose, this month, of how to navigate the virtual *ocean* of resume guidance, tips, and templates out there, and decide which advice is the best advice. Since most of us have been in this position before, there were lots of helpful suggestions:
 
-* There is no 'right way' to write a resume. All of the mechanical and organization rules are dependant on you, your background, the job listing, the target company, etc.
+* There is no 'right way' to write a resume. All of the mechanical and organization rules are dependent on you, your background, the job listing, the target company, etc.
```

Comment:
Consider changing "are dependent on you" to "depend on you".

Either way is technically correct, but the alternative is more concise.

Reply:
Did exactly that, thanks!